server/src/modules/Question/question.service.ts

```diff
@@ -6,73 +6,75 @@ import { QuestionDto } from './dto/question.dto';
 
 @Injectable()
 export class QuestionService {
+  constructor(
+    @InjectRepository(Question)
+    private readonly questionRepository: Repository<Question>,
+  ) {}
 
-    constructor(
-        @InjectRepository(Question)
-        private readonly questionRepository: Repository<Question>,
-    ) { }
-
-    async findAll(): Promise<Question[]> {
-        try {
-            return await this.questionRepository.find();
-        } catch (err) {
-            return err;
-        }
+  async findAll(): Promise<Question[]> {
+    try {
+      return await this.questionRepository.find();
+    } catch (err) {
+      return err;
     }
+  }
 
-    // todo it's a temporary solution and will be implemented in #23/24 stories
-    async getRandom(): Promise<Question> {
-        try {
-            return await this.questionRepository.createQueryBuilder()
-                .orderBy('RAND()')
-                .getOne();
-        } catch (err) {
-            return err;
-        }
+  // todo it's a temporary solution and will be implemented in #23/24 stories
+  async getRandom(): Promise<Question> {
+    try {
+      return await this.questionRepository
+        .createQueryBuilder()
+        .orderBy('RAND()')
+        .getOne();
+    } catch (err) {
+      return err;
     }
+  }
 
-    async findById(id: string): Promise<Question> {
-        try {
-            return await this.questionRepository.findOne(id);
-        } catch (err) {
-            return err;
-        }
+  async findById(id: string): Promise<Question> {
+    try {
+      return await this.questionRepository.findOne(id);
+    } catch (err) {
+      return err;
     }
+  }
 
-    async insert(question: QuestionDto): Promise<QuestionDto> {
-        const newQuestion = new Question();
+  async insert(question: QuestionDto): Promise<QuestionDto> {
+    const newQuestion = new Question();
 
-        Object.keys(question).forEach((key) => {
-            newQuestion[key] = question[key];
-        });
+    Object.keys(question).forEach(key => {
+      newQuestion[key] = question[key];
+    });
 
-        try {
-            return await this.questionRepository.save(newQuestion);
-        } catch (err) {
-            return err;
-        }
+    try {
+      return await this.questionRepository.save(newQuestion);
+    } catch (err) {
+      return err;
```

Comment:
Please, handle errors appropriately (check nest's `HttpException` wrapper), this one will throw 500 in case of unsuccessful db validation